tests/AutoReview/ProjectCodeTest.php

```diff
@@ -255,6 +256,42 @@ public function testThatDataProvidersAreCorrectlyNamed($testClassName, $dataProv
         ));
     }
 
+    /**
+     * @dataProvider provideClassesWherePregFunctionsAreForbiddenCases
+     *
+     * @param string $className
+     */
+    public function testThereIsNoPregFunctionUsedDirectly($className)
+    {
+        $rc = new \ReflectionClass($className);
+        $tokens = Tokens::fromCode(file_get_contents($rc->getFileName()));
```

Comment:
Did you try running this test with Xdebug enabled? I think it might be very slow, maybe worth skipping it in such case like e.g. in [`ProjectCodeTest::provideDataProviderMethodNameCases()`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.10.0/tests/AutoReview/ProjectCodeTest.php#L338).

Reply:
Yeah, was slow - I'll disable them when Xdebug is enabled.

What about going further and disable whole AutoReview for Xdebug?

Reply:
If others are really slow, why not, maybe in another PR. But for me it was only super slow for the already disabled test, which uses `Tokens::fromCode()`. This is why I commented on this specific line.